Tribler/Core/Modules/restapi/torrentinfo_endpoint.py

```diff
@@ -106,8 +106,12 @@ def on_lookup_error(failure):
                 request.setResponseCode(http.INTERNAL_SERVER_ERROR)
                 return json.dumps({"error": "error while decoding torrent file"})
         elif uri.startswith('http'):
-            def _on_loaded(metadata):
-                metainfo_deferred.callback(bdecode(metadata))
+            def _on_loaded(response):
+                if response.startswith('magnet:?xt=urn:btih:'):
```

Comment:
You should be careful with this. It could be that the magnet link does not start with the `btih` element but instead the name. I would rather have something like:

```
if response.startswith('magnet:'):
```